R/input_checking_functions.R

```diff
@@ -272,3 +272,34 @@ check_boolean <- function(args_to_check) {
     }
     return(error_message)
 }
+
+# Checks that the `time` variable is ordered, strictly increasing, and evenly spaced with the `timestep`.
+# The check is that for all n, dt = t[n] - t[n-1] up to error tolerance due to
+# inexact floating point arithmetic.
+check_time_is_sequential <- function(drivers, differential_modules, rtol = sqrt(.Machine$double.eps)){
```

Comment:
The input arguments could be moved to separate lines to avoid exceeding 80 characters.